sample-preprod-maker-bot-config-gens-v2.yaml

```diff
@@ -0,0 +1,108 @@
+    pcc_respite_delay_factor: 0.2
+    # Relaxation time (in cycles) to return to normal after piercing threshold1.
+    pcc_after_exit_relax_aim1:  4
+    # Waiting time (in cycles) to start warning after piercing threshold1.
+    pcc_after_exit_worse_max1:  5
+    # Relaxation time (in cycles) to return to normal after piercing threshold2.
+    pcc_after_exit_relax_aim2:  8
+    # Relative weights in the computation of average price. Example: 50% (maestro) + 50% (taptools).
+    pcc_prices_providers_weights:
+      - 50  # Maestro
+      - 50  # Taptools
+  # Configuration parameters for each prices provider
+  pc_prices_provider_cfgs:
+    - tag: MaestroPPC
+      contents:
+        # Maestro API key. Fetching prices from https://docs.gomaestro.org/DefiMarketAPI/mkt-dex-ohlc endpoint requires a paid subscription.
+        mc_api_key: <<MAESTRO_TOKEN>>
+        # Maestro resolution (overriding common resolution).  Options: 1m, 5m, 15m, 30m, 1h, 4h, 1d, 1w, 1mo.
+        # mc_resolution_override: 
+        mc_dex: minswap
+        # Alternative pair (for testing only).
+        mc_pair_override:
+          mpo_pair: "ADA-GENS"
+          mpo_commodity_is_first: false
+    - tag: TaptoolsPPC
+      contents:
+        # Taptools API key.
+        ttc_api_key: <<TAPTOOLS_TOKEN>>
+        # Taptools resolution (overriding common resolution). Options: 3m, 5m, 15m, 30m, 1h, 2h, 4h, 12h, 1d, 3d, 1w, 1mo
+        # ttc_resolution_override:
+        # Alternative asset with precision (for testing only).
+        ttc_pair_override:
+          ttpo_asset: dda5fdb1002f7389b33e036b6afee82a8189becb6cba852e8b79b4fb.0014df1047454e53  # example: GENS
+          ttpo_precision: 6
+# Please see description following this sample for details regarding `mbc_strategy_config` field.
+mbc_strategy_config:
+  sc_spread:
+    # Can simple write
+    # numerator: 1
+    # denominator: 100
+    # To specify same spread for both sell and buy side. Or can individually specify for each like done below.
+    sell_side_spread:
+      numerator: 1
+      denominator: 100
+    buy_side_spread:
+      numerator: 2
+      denominator: 100
+  sc_token_volume:
+    tv_sell_min_vol: 1000000000
+    tv_buy_min_vol: 2000000000
+    tv_sell_budget: 3500000000
+    tv_buy_budget: 6600000000
+    tv_sell_vol_threshold: 10000000000
+    tv_buy_vol_threshold: 20000000000
+  sc_price_check_product: 9
+  sc_cancel_threshold_product: 4
+  sc_preemptive_cancel_spread_ratio:
+    numerator: 1
+    denominator: 10
+# Token details.
+mbc_token:
+  # Asset class represented as concatenation of currency symbol and asset name with dot separator.
+  # Below sample is of `tGENS` test token.
+  ac: c6e65ba7878b2f8ea0ad39287d3e2fd256dc5c4160fc19bdf4c4d87e.7447454e53
+  # Registered decimal places for the token. As an examples, for `ADA` it is 6 and for `SNEK` it is 0.
+  precision: 6
+# Boiler plate for Pre-production testnet.
+mbc_fp_nft_policy: compiled-scripts/minting-policy
```

Comment:
This boiler plate was removed recently and so has to be omitted for.

Reply:
Done.